packages/components/pie-card-container/src/index.ts

```diff
@@ -58,12 +66,42 @@ export class PieCardContainer extends LitElement implements CardContainerProps {
                 target=${this.target || nothing}
                 rel=${this.rel || nothing}
                 aria-label=${this.aria?.label || nothing}
-                aria-disabled=${this.disabled ? 'true' : 'false'}>
+                aria-disabled=${this.disabled ? 'true' : 'false'}
+                style=${paddingCSS ? `padding: ${paddingCSS}` : nothing}>
                     <slot></slot>
                 </div>
             </a>`;
     }
 
+    /**
+     * Generates padding for the component based on `padding` or `paddingX` values passed
+     * by the consumer.
+     *
+     * @private
+     */
+    private generatePaddingCSS (): string {
```

Comment:
I think potentially we could via a mixin / helper although i'm not sure how it would work here in this file, we might need to pass a class in and use that as a hook to apply those styles.

Reply:
Shall we create a ticket for this?

Reply:
I'm ok with it living here for now and if we ever need it elsewhere, pulling it out at that stage?